src/main/java/pvs/app/controller/GithubApiController.java

```diff
@@ -88,6 +88,44 @@ public GithubApiController(GithubApiService githubApiService, GithubCommitServic
         }
     }
 
+    @GetMapping("/github/commits/branchName/{repoOwner}/{repoName}")
+    public ResponseEntity<List<String>> getBranchesName(@PathVariable("repoOwner") String repoOwner, @PathVariable("repoName") String repoName) {
+        Date lastUpdate;
+        GithubCommitDTO githubCommitDTO = githubCommitService.getLastCommit(repoOwner, repoName);
+        if (null == githubCommitDTO) {
+            Calendar calendar = Calendar.getInstance();
+            calendar.set(1970, Calendar.JANUARY, 1);
+            lastUpdate = calendar.getTime();
+        } else {
+            lastUpdate = githubCommitDTO.getCommittedDate();
+        }
```

Comment:
Suggest simplify the statement.

```suggestion
final Date lastUpdate = githubCommitDTO == null ? Date.from(Instant.ofEpochSecond(0)) : githubCommitDTO.getCommittedDate();
```